paimon-core/src/main/java/org/apache/paimon/catalog/Identifier.java

```diff
@@ -87,21 +88,41 @@ public static Identifier fromString(String fullName) {
     }
 
     public static Identifier fromPath(Path tablePath) {
-        return fromPath(tablePath.getPath());
+        return fromPath(tablePath, false);
     }
 
     public static Identifier fromPath(String tablePath) {
+        return fromPath(tablePath, false);
+    }
+
+    public static Identifier fromPath(Path tablePath, boolean ignoreIfUnknownDatabase) {
+        return fromPath(tablePath.getPath(), ignoreIfUnknownDatabase);
+    }
+
+    public static Identifier fromPath(String tablePath, boolean ignoreIfUnknownDatabase) {
```

Comment:
Consider `Identifier` is a public api, can we just put these logical in [SchemaManager.java](https://github.com/apache/incubator-paimon/pull/1051/files#diff-fdc47daefd9d3a55f45fde8385a8be476f6834c30649f7f5bae98cbe90f15dbb)?

Reply:
done.